.travis.yml

```diff
@@ -1,23 +1,40 @@
 language: scala
+
 scala:
+  - 2.12.2-bin-typelevel-4
+  - 2.11.11-bin-typelevel-4
```

Comment:
I can't think of anywhere that blaze benefits featurewise.  It does make it easier to eventually spin off an http4s organization plugin to make repos consistent.  This becomes more compelling if we go polyrepo with the support modules in http4s.  Also, the more projects build with typelevelc, the more we're helping to test it.

I can argue both ways flimsily, but am likely to side with whoever pounds on the table the loudest.